site-manage/site-manage-tool/tool/src/webapp/js/site-manage.js

```diff
@@ -685,11 +685,17 @@ var setupCategTools = function () {
     var noMsg = $("#removalAlertNoMessage").val();
     $("<li id=\"alertBox\"><span>" + msg + "</span><span><a href=\"#\" id=\"alertBoxYes\">" + yesMsg + "</a>&nbsp;|&nbsp;<a href=\"#\" id=\"alertBoxNo\">" + noMsg + "</a></span></li>").insertAfter("#" + target);
 
-    $("#alertBox a#alertBoxYes").on("click", function () {
-
-      $(this).closest("li").prev("li").remove();
-      $("#" + target +"_wrap").find("input").attr("checked", false).end().find("label").css("font-weight", "normal");
-      $("#alertBox").remove();
+    $('#alertBox a#alertBoxYes').on('click', function () {
+      $(this).closest('li').prev('li').remove();
+      var checkboxInput = document.getElementById(target.replace('_', '.'));
+      if (checkboxInput != null) {
+        checkboxInput.checked = false;
+        var checkboxLabel = checkboxInput.closest('div').querySelector('label');
+        if (checkboxLabel != null) {
+          checkboxLabel.style.fontWeight = 'normal';
```

Comment:
styling via js is generally not a good thing to do, think of inline styles in html!

Reply:
I fixed what is broken, respecting what was implemented.

What should we do here in your opinion? That line restores the font weight in the tool list:

![a_2](https://user-images.githubusercontent.com/8653754/77778523-68bb7900-7051-11ea-83bf-60617db981d0.jpg)

Reply:
Yeah, looks like [Bootstrap 4](https://getbootstrap.com/docs/4.0/utilities/text/#font-weight-and-italics) has these but doesn't look [like there's font-weight classes in 3.3.x](https://stackoverflow.com/a/27584395/3708872) so you'd have to workaround.

I don't *see* classes defined yet to be similar to these. Maybe something to clean up in the future if/when we upgrade Bootstrap.